package.json

```diff
@@ -44,5 +44,8 @@
     "husky": "^9.1.7",
     "nx": "^20.2.1",
     "prettier": "^3.4.2"
+  },
+  "dependencies": {
+    "polkadot-api": "1.8.0"
```

Comment:
This can be removed. PAPI is a peer dependency and should be updated as a devDependency in package.json

Reply:
I think there are several places that needed to be updated, you can just do a string search for "polkadot-api"

Reply:
> I think there are several places that needed to be updated, you can just do a string search for "polkadot-api"

Yeah, I found other `package.json`s that reference it. THose versions should be updated, right?

Reply:
I'm on it

Reply:
> Yeah, I found other `package.json`s that reference it. THose versions should be updated, right?

A few of those I found are examples. Do i leave those under `dependencies`? The ones in `packages` are to be left in `devDependencies`?

Reply:
Yes should be good leaving them where they are.

Reply:
Please remove this